memory-viz/src/automate.ts

```diff
@@ -97,7 +119,7 @@ function drawAutomatedStackFrames(stack_frames: DrawnEntity[], configuration) {
         min_required_height = height + min_required_height;
     }
 
-    required_width += configuration.padding;
+    required_width += configuration.left_margin;
```

Comment:
Hi Professor, this was changed because object boxes would overlap with stack frame objects if the left margin values were large, and this seemed to fix the issue (I think the object boxes were being drawn at the wrong x-coordinates). Should I revert it?

Reply:
Yes, if it's a separate bug then you can investigate and fix it in a separate PR